spec/controllers/v2/users/sessions_controller_spec.rb

```diff
@@ -33,5 +33,31 @@
         it { expect(response_hash['error']).not_to be_empty }
       end
     end
+
```

Comment:
I wanted to do that in #62, but we were not sure, if this is the right place for it. @eugenk what would you suggest?